packages/web-socket-browser/src/ws-browser-bus-client.ts

```diff
@@ -0,0 +1,148 @@
+import { IpcBusClientImpl } from '@electron-common-ipc/universal';
+
+import { ReconnectionTimeoutMs } from './constants';
+
+import type {
+    ClientConnectOptions,
+    ClientCloseOptions,
+    EventEmitterLike,
+    IpcBusListener,
+    IpcBusTransport,
+    BusMessagePort,
+    IpcBusPeer,
+    IpcBusRequestResponse,
+    Logger } from '@electron-common-ipc/universal';
```

Comment:
Looks like formatting is wrong, could yo format the document? You can use the eslint-pettier extension on any other formatting tool compatible with prettier